en/docs/develop/customizing-identity-server-uis.md

```diff
@@ -260,6 +253,3 @@ Add the `product-footer.jsp` and `header.jsp` files to the extensions folder as
     <span class="figcaption_hack">Customized Self signup page</span>
 
 
-!!! note  
-    One advantage of this approach is that WUM updates will not affect your UI changes and you need not worry about manually adding your changes. 
-    
```

Comment:
We might need this part since this generally speaks about the process used in the customizing of uis.

Reply:
fixed e6d488f519594296e49be361cf323f803df4df8b